Zend/tests/serializable_deprecation.phpt

```diff
@@ -20,4 +20,4 @@ class D extends A implements I {
 
 ?>
 --EXPECTF--
-Deprecated: The Serializable interface is deprecated. Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary) in %s on line %d
+Deprecated: %s implements the Serializable interface, which is deprecated. Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary) in %s on line %d
```

Comment:
The PHPT files were all changed by a single search/replace. I assume the class name is as irrelevant as the filename and line number, which also use patterns.

That said, if it makes sense to have at least one assertion for the message including the offending class name perhaps `Zend/tests/serializable_deprecation.phpt` is the test in which to do so. Let me know if so.